### PR TITLE
Dynaconf conversion for http_proxy and oscap section

### PR DESCRIPTION
**Pull request by jameerpathan111**

### CODE REVIEW

File: conf/http_proxy.yaml.template [hunk 1]
```diff
@@ -0,0 +1,5 @@
+HTTP_PROXY:
+  UN_AUTH_PROXY_URL: http://proxy-01.example.com:3423
+  AUTH_PROXY_URL: http://proxy-02.example.com:3423
```
Comment (akhil-jha) [inline conf/http_proxy.yaml.template:3]: I think you could comment this key unless it has to be this value.

File: conf/oscap.yaml.template [hunk 2]
```diff
@@ -0,0 +1,3 @@
+OSCAP:
+  CONTENT_PATH: /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml
+  TAILORING_PATH: http://server.example.com/pub/ssg-rhel7-ds-tailoring.xml
```
Comment (akhil-jha) [inline conf/oscap.yaml.template:3]: same.

**Review by akhil-jha — CHANGES_REQUESTED** [review]

**Review by omkarkhatavkar — CHANGES_REQUESTED** [review]
Please provide the test result as well, make sure that you disable the robottelo.properties file while running the test

**Comment by jameerpathan111** [comment]
Created oscap.yaml and http_proxy.yaml, commented oscap and http_proxy sections in robottelo.properties, and then ran the following tests.
Test result:
```
$ pytest tests/foreman/cli/test_oscap_tailoringfiles.py 
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jpathan/projects/robottelo, configfile: pyproject.toml
plugins: ibutsu-1.13, services-2.2.1, mock-3.5.1, reportportal-5.0.7, xdist-2.2.1, forked-1.3.0
collected 12 items / 4 deselected / 8 selected

tests/foreman/cli/test_oscap_tailoringfiles.py ......s.                  [100%]

============ 7 passed, 1 skipped, 4 deselected in 132.34s (0:02:12) ============
```
```
$ pytest tests/foreman/cli/test_product.py -k test_positive_assign_http_proxy_to_products | tee http_proxy.log 
============================= test session starts ==============================
platform linux -- Python 3.8.6, pytest-6.2.2, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/jpathan/projects/robottelo, configfile: pyproject.toml
plugins: ibutsu-1.13, services-2.2.1, mock-3.5.1, reportportal-5.0.7, xdist-2.2.1, forked-1.3.0
collected 7 items / 6 deselected / 1 selected

tests/foreman/cli/test_product.py .                                      [100%]

=============================== warnings summary ===============================
tests/foreman/cli/test_product.py::test_positive_assign_http_proxy_to_products
tests/foreman/cli/test_product.py::test_positive_assign_http_proxy_to_products
  /home/jpathan/projects/env_robottelo_38/lib/python3.8/site-packages/urllib3/connectionpool.py:1013: InsecureRequestWarning: Unverified HTTPS request is being made to host 'dhcp-2-114.vms.sat.rdu2.redhat.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
    warnings.warn(

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========== 1 passed, 6 deselected, 2 warnings in 260.47s (0:04:20) ============
```

**Review by akhil-jha — APPROVED** [review]

File: conf/oscap.yaml.template [hunk 3]
```diff
@@ -0,0 +1,3 @@
+OSCAP:
+  CONTENT_PATH: /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml
```
[the remark this answers is not in the document]
Comment (jameerpathan111) [reply conf/oscap.yaml.template]: done

**Review by mshriver — APPROVED** [review]

**Comment by mshriver** [comment]
All previous comments and requests for test results have been met.

**Pull request merged**